code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -298,6 +298,10 @@
 	scream_verb = "shrieks"
 	male_scream_sound = 'sound/voice/shriek1.ogg'
 	female_scream_sound = 'sound/voice/shriek1.ogg'
+	male_cough_sounds = 'sound/voice/shriek1.ogg'
```

Comment:
This needs to be a list of a single entry, since `pick` will be called on this value to choose which sound to play.